src/test/java/org/candlepin/subscriptions/rhmarketplace/RhMarketplacePayloadMapperTest.java

```diff
@@ -81,7 +81,6 @@ void testUsesMetricIdRatherThanUom() {
 
     rhMarketplacePayloadMapper.produceUsageMeasurement(mockUsage);
     verify(mockUsage).getMetricId();
-    verify(mockUsage, never()).getUom();
```

Comment:
Is this test still relevant now that `uom` is no longer in the billable_usage schema?

Reply:
Let's remove it then.